App/Models/TextileTypes.ts

```diff
@@ -220,7 +220,8 @@ export interface ILocalPhotoResult {
   modificationDate: string,
   orientation: number,
   path: string,
-  uri: string
+  uri: string,
+  canDelete: boolean
```

Comment:
This is now part of the payload that comes from the native layer. This way, ios/android can handle it differently as is probably the case, and if ever in the future one changes it doesn't need to be hunted down.

Reply:
Yeh nice, good call